postgres-appliance/Dockerfile

```diff
@@ -3,9 +3,12 @@ ARG TIMESCALEDB=1.7.4
 ARG TIMESCALEDB_LEGACY=1.7.4
 ARG DEMO=false
 ARG COMPRESS=false
+ARG ADDITIONAL_LOCALES=
```

Comment:
```suggestion
```
Global ARG is only required when we want to use it in multiple stages.

Reply:
Removed global ARG